bazelisk.go

```diff
@@ -152,7 +160,16 @@ func resolveLatestVersion(bazeliskHome string, offset int) (string, error) {
 	return versions[len(versions)-1-offset].Original(), nil
 }
 
-func resolveVersionLabel(bazeliskHome, bazelVersion string) (string, error) {
+func resolveVersionLabel(bazeliskHome, bazelVersion string) (string, bool, error) {
```

Comment:
Can you use named return values here (or add a quick comment what this function returns)? I didn't immediately understand what the bool is for.